src/m365/planner/commands/plan/plan-details-get.ts

```diff
@@ -0,0 +1,177 @@
+import { Logger } from '../../../../cli';
+import { CommandOption} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import Utils from '../../../../Utils';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  planId?: string;
+  planTitle?: string;
+  ownerGroupId?: string;
+  ownerGroupName?: string;
+}
+
+class PlannerPlanDetailsGetCommand extends GraphItemsListCommand<any> {
```

Comment:
Since we're retrieving just one item, let's use the `GraphCommand` base class

Reply:
Done:)